lib/repeatable.js

```diff
@@ -58,7 +58,9 @@ module.exports = function(Queue) {
               _.defaultsDeep(
                 {
                   repeat: {
-                    count: currentCount
+                    count: currentCount,
+                    // Do NOT save immediately flag to true
```

Comment:
why is this needed?

Reply:
I re-read my code, and like I suggest, I need to replace this by a `delete opts.repeat.immediately;` after the getNextMillis call.